### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by tylerliuyang**

### CODE REVIEW

**Comment by jsu21ges** [comment]
hmm yes, I see

**Comment by codingzencc** [comment]
@tylerliuyang can you please remove assignment 3 as I already added it last week. Also remove the .beam file associated with it and add .beam to .gitignore.

File: assignment_4/main.beam
Comment (codingzencc) [inline assignment_4/main.beam:1]: remove this file

File: assignment_3/main.erl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+-module(main).
```
Comment (codingzencc) [inline assignment_3/main.erl]: already added assignment 3 last week, so please remove it.

**Pull request merged**